.claude/skills/twelve-factor-methodology/scripts/agents-runner.sh

```
@@ -0,0 +1,96 @@
+
+echo "Found ${#AGENTS[@]} agents:"
+for agent in "${AGENTS[@]}"; do
+  echo "  - $(basename "$agent")"
+done
+
+# ============================================================================
+# Run agents in parallel with concurrency control
+# ============================================================================
+run_agent() {
+  local agent="$1"
+  local agent_name=$(basename "$agent" .sh)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)